File: BOJ/Java/단계별로_풀어보기/B_01_입출력과_사칙연산/BOJ_10869.java
package 단계별로_풀어보기.B_01_입출력과_사칙연산;

/**
 * BOJ 10869 : 사칙연산
 * 
 * @author kdgyun
 * {@link https://st-lab.tistory.com}
 * {@link https://github.com/kdgyun}
 *
 */

import java.util.Scanner;

public class BOJ_10869 {

	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);

		String e = in.next();
		System.out.println(e);
		String aa = in.nextLine();

		System.out.println(aa);
		int A = in.nextInt();
		int B = in.nextInt();

		System.out.println(A + B);
		System.out.println(A - B);
		System.out.println(A * B);
		System.out.println(A / B);
		System.out.println(A % B);

		in.close();

	}

}

File: BOJ/CPP/단계별로_풀어보기/01_입출력과_사칙연산/BOJ_10869.cpp
/*
    BOJ 10869 : 사칙연산

    author : kdgyun
    link https://st-lab.tistory.com
    link https://github.com/kdgyun
    Created by kdgyun
    Copyright © 2021 kdgyun. All rights reserved.
*/

#include <stdio.h>

int main(int argc, char const *argv[]) {
    int a, b;

    scanf("%d %d", &a, &b);
    printf("%d\n", a + b); // 덧셈
    printf("%d\n", a - b); // 뺄셈
    printf("%d\n", a * b); // 곱셈
    printf("%d\n", a / b); // 나눗셈 (몫)
    printf("%d\n", a % b); // 나눗셈 (나머지)
    return 0;
}
